recipes/sdl_mixer/cmake/conanfile.py

```diff
@@ -0,0 +1,212 @@
+        # OPUS
+        tc.variables["SDL2MIXER_OPUS"] = self.options.opus
+        # VORBIS
+        if self.options.vorbis == "stb":
+            tc.variables["SDL2MIXER_VORBIS"] = "STB"
+        elif self.options.vorbis == "vorbisfile":
+            tc.variables["SDL2MIXER_VORBIS"] = "VORBISFILE"
+        elif self.options.vorbis == "tremor":
+            tc.variables["SDL2MIXER_VORBIS"] = "TREMOR"
+        else:
+            tc.variables["SDL2MIXER_VORBIS"] = False
+
+        # TODO: add support for dynamic loading of dependencies
+        tc.variables["SDL2MIXER_DEPS_SHARED"] = False
+        tc.variables["SDL2MIXER_FLAC_LIBFLAC_SHARED"] = False
+        tc.variables["SDL2MIXER_MIDI_FLUIDSYNTH_SHARED"] = False
+        tc.variables["SDL2MIXER_MIDI_TIMIDITY_SHARED"] = False
+        tc.variables["SDL2MIXER_MOD_MODPLUG_SHARED"] = False
+        tc.variables["SDL2MIXER_MOD_XMP_SHARED"] = False
+        tc.variables["SDL2MIXER_MP3_MPG123_SHARED"] = False
+        tc.variables["SDL2MIXER_OGG_SHARED"] = False
+        tc.variables["SDL2MIXER_OPUS_SHARED"] = False
+        tc.variables["SDL2MIXER_SNDFILE_SHARED"] = False
+        tc.variables["SDL2MIXER_VORBIS_TREMOR_SHARED"] = False
+        tc.variables["SDL2MIXER_VORBIS_VORBISFILE_SHARED"] = False
+        tc.generate()
+
+        deps = CMakeDeps(self)
+        deps.set_property("flac", "cmake_file_name", "FLAC")
+        deps.set_property("flac", "cmake_target_name", "FLAC")
+        deps.set_property("fluidsynth", "cmake_file_name", "FluidSynth")
+        deps.set_property("fluidsynth", "cmake_target_name", "FluidSynth::FluidSynth")
+        deps.set_property("libxmp", "cmake_file_name", "libxmp")
+        deps.set_property("libxmp", "cmake_target_name", "libxmp::libxmp")
+        deps.set_property("libmodplug", "cmake_file_name", "modplug")
+        deps.set_property("libmodplug", "cmake_target_name", "modplug::modplug")
+        deps.set_property("mpg123", "cmake_file_name", "MPG123")
+        deps.set_property("mpg123", "cmake_target_name", "MPG123::mpg123")
+        deps.set_property("opusfile", "cmake_file_name", "opusfile")
+        deps.set_property("opusfile", "cmake_target_name", "opusfile::opusfile")
+        deps.set_property("tremor", "cmake_file_name", "tremor")
+        deps.set_property("tremor", "cmake_target_name", "tremor::tremor")
+        deps.set_property("vorbis", "cmake_file_name", "vorbisfile")
+        deps.set_property("vorbis::vorbisfile", "cmake_target_name", "vorbisfile::vorbisfile")
+
+        deps.generate()
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, "LICENSE.txt", self.source_folder, os.path.join(self.package_folder, "licenses"))
+        cmake = CMake(self)
+        cmake.install()
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
```

Comment:
On Windows, it seems to be `self, os.path.join(self.package_folder, "cmake")`. The suggestion is not tested:
```suggestion
        if self.settings.os == "Windows":
            rmdir(self, os.path.join(self.package_folder, "cmake"))
        else:
            rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
```

From the logs:
Linux:
> -- Install configuration: "Release"
> -- Installing: /home/conan/w/prod-v2/bsr/70187/fbaed/p/b/sdl_md2106973f57d3/p/lib/libSDL2_mixer.a
> -- Installing: /home/conan/w/prod-v2/bsr/70187/fbaed/p/b/sdl_md2106973f57d3/p/include/SDL2/SDL_mixer.h
> -- Installing: /home/conan/w/prod-v2/bsr/70187/fbaed/p/b/sdl_md2106973f57d3/p/lib/cmake/SDL2_mixer/SDL2_mixerConfig.cmake


Windows:
> -- Installing: C:/J2/w/prod-v1/bsr@2/84698/ceffe/.conan/data/sdl_mixer/2.6.3/_/_/package/51188b1528b7de4f0f4ac2754c459259f18a23cd/lib/SDL2_mixer-static.lib
> -- Installing: C:/J2/w/prod-v1/bsr@2/84698/ceffe/.conan/data/sdl_mixer/2.6.3/_/_/package/51188b1528b7de4f0f4ac2754c459259f18a23cd/include/SDL2/SDL_mixer.h
> -- Installing: C:/J2/w/prod-v1/bsr@2/84698/ceffe/.conan/data/sdl_mixer/2.6.3/_/_/package/51188b1528b7de4f0f4ac2754c459259f18a23cd/cmake/SDL2_mixerConfig.cmake

After checking, the code in SDL_mixer that set this variable for Windows is: https://github.com/libsdl-org/SDL_mixer/blob/SDL2/CMakeLists.txt#L910. I'm not sure of the reason but if there is a if in the code, there should be a reason so the workaround in conan may make sense...